pgp-plugin/src/main/scala/com/typesafe/sbt/pgp/PgpSettings.scala

```diff
@@ -161,16 +161,25 @@ object PgpSettings {
         checksums = (checksums in publish).value,
         logging = ivyLoggingLevel.value)
     },
-    publishSigned := Classpaths.publishTask(publishSignedConfiguration, deliver).value,
+    publishSigned := publishSignedTask(publishSignedConfiguration, deliver).value,
     publishLocalSignedConfiguration := {
       Classpaths.publishConfig(
         signedArtifacts.value,
         Some(deliverLocal.value),
         (checksums in publishLocal).value,
         logging = ivyLoggingLevel.value)
     },
-    publishLocalSigned := Classpaths.publishTask(publishLocalSignedConfiguration, deliver).value
+    publishLocalSigned := publishSignedTask(publishLocalSignedConfiguration, deliver).value
   )
+
+  def publishSignedTask(config: TaskKey[PublishConfiguration], deliverKey: TaskKey[_]): Def.Initialize[Task[Unit]] =
+    Def.taskDyn {
+      val s = streams.value
+      val skp = ((skip in publish) ?? false).value
+      if (skp) Def.task { s.log.debug("Skipping publishSigned") }
```

Comment:
Instead of "Skipping publishSigned" can we say "`publishSigned` is skipped for project `foo`"?